opennms-base-assembly/src/main/filtered/etc/snmp-graph.properties.d/juniper-graph.properties

```diff
@@ -94,6 +95,29 @@ report.juniper.temp.command=--title="Current Temperature" \
  GPRINT:val3:MAX:"Max  \\: %8.2lf %s\\n"
 
 #####
+##### Juniper SRX reports
+#####
+
+report.srx.spuFlowMonitor.name=Juniper SRX Flows
+report.srx.spuFlowMonitor.columns=jnxJsSPUMonitoringCurrentFlowSession,jnxJsSPUMonitoringMaxFlowSession
+report.srx.spuFlowMonitor.type=jnxJsSPUMonitoringObjectsTable
+report.srx.spuFlowMonitor.propertiesValues=jnxJsSPUMonitoringNodeDescr
+report.srx.spuFlowMonitor.command=--title="Sessions used" \
+ --vertical-label="Sessions" \
+ DEF:currentSessions={rrd1}:jnxJsSPUMonitoringC:AVERAGE \
```

Comment:
yep, tha's right: Inside the jrb file the datasource's name is trimmed to 19 characters :-)

I can fix that in my PR, no problem.